cartographer/mapping/probability_values.cc

```diff
@@ -42,7 +42,9 @@ std::unique_ptr<std::vector<float>> PrecomputeValueToBoundedFloat(
   auto result = absl::make_unique<std::vector<float>>();
   // Repeat two times, so that both values with and without the update marker
   // can be converted to a probability.
-  for (int repeat = 0; repeat != 2; ++repeat) {
+  const int repeat_times = 2;
+  result->reserve(repeat_times * kNumberOfValues);
```

Comment:
nit: Since you are here, I think renaming kNumberOfValues -> kValueCount reads a bit better.

Reply:
Done but maybe you prefer that I open a new PR for this change?

Reply:
Well, I will do it in a following PR.

Reply:
@damienrg  Sorry, noticed it too late. It takes quite long to submit a PR here on Github. I think for such small PRs it makes sense to keep it in one change to keep submitting code quicker and not resolving conflicts all the time. Keep it in two PRs since you have already done that. But for the future PRs, just combine.